src/utilities/filterFunctions/getSortedPriceProducts.js

```diff
@@ -0,0 +1,15 @@
+export const getSortedPriceProducts = (products, sortByPrice) => {
+  if (sortByPrice === "PRICE_LOW_TO_HIGH") {
+    return products.sort(
+      (firstItem, secondItem) =>
+        firstItem.discountedPrice - secondItem.discountedPrice
+    );
+  }
+  if (sortByPrice === "PRICE_HIGH_TO_LOW") {
+    return products.sort(
+      (firstItem, secondItem) =>
+        secondItem.discountedPrice - firstItem.discountedPrice
+    );
+  }
```

Comment:
since sort is not a pure function, you should destructure the `products` before sorting in this case
```js
[...products].sort( (firstItem, secondItem) => firstItem.discountedPrice - secondItem.discountedPrice)
```